spec/provider/deis_spec.rb

```diff
@@ -67,6 +67,15 @@
       expect(provider.context).to receive(:shell).with(
         './deis git:remote --app=example'
       ).and_return(true)
+
+      expect(provider.context).to receive(:shell).with(
+        /grep -c 'PTY allocation request failed'/
```

Comment:
It is, but since most of the rest of it was dynamic and this part was pretty unique I wasn't concerned with looking for the whole line. I can come up with some regex to match it all if that is still preferred though.